account-kit/react/src/experimental/hooks/useGetCallsStatus.ts

```diff
@@ -8,8 +8,8 @@ import { useAlchemyAccountContext } from "../../hooks/useAlchemyAccountContext.j
 import { ReactLogger } from "../../metrics.js";
 
 export type UseGetCallsStatusParams = {
-  client?: GetSmartWalletClientResult<Address>;
-  callId?: Hex;
+  client: GetSmartWalletClientResult<Address> | undefined;
```

Comment:
i think it makes more sense for the `client` to not be optional, b/c it is technically required for the hook for do anything. it can only be `undefined` during loading states.